### PR TITLE
[ODS-4881] Fixes after Dockerhub updates

### PR DESCRIPTION
**Pull request by jamessmckay**

### CODE REVIEW

File: Web-Ods-AdminApp/Alpine/pgsql/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ COPY appsettings.template.json /app/appsettings.template.json
 COPY run.sh /app/run.sh
 COPY log4net.config /app/log4net.txt
 
-RUN apk --no-cache add unzip=6.0-r8 dos2unix=7.4.2-r0 bash=5.1.0-r0 gettext=0.20.2-r2 postgresql-client=13.2-r0 icu=67.1-r2 && \
+RUN apk --no-cache add unzip=6.0-r8 dos2unix=7.4.2-r0 bash=5.1.0-r0 gettext=0.20.2-r2 postgresql-client=13.2-r0 jq=1.6-r1 icu=67.1-r2 && \
```
Comment (vimayya) [inline Web-Ods-AdminApp/Alpine/pgsql/Dockerfile:24]: @jamessmckay we probably need the same fix on mssql Dockerfile

Comment (jamessmckay) [reply Web-Ods-AdminApp/Alpine/pgsql/Dockerfile:24]: i just double checked i had it correct in the mssql version
```
RUN apk --no-cache add unzip=6.0-r8 dos2unix=7.4.2-r0 bash=5.1.0-r0 gettext=0.20.2-r2 postgresql-client=13.2-r0 jq=1.6-r1 icu=67.1-r2 && \
    wget -O /app/AdminApp.zip https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_apis/packaging/feeds/EdFi/nuget/packages/EdFi.Suite3.ODS.AdminApp.Web/versions/${VERSION}/content && \
    unzip /app/AdminApp.zip -d /app && \
    rm -f /app/AdminApp.zip && \
    cp /app/log4net.txt /app/log4net.config && \
    dos2unix /app/*.json && \
    dos2unix /app/*.sh && \
    dos2unix /app/log4net.config && \
    chmod 700 /app/*.sh -- ** && \
    rm -f /app/*.exe
```

File: README.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ An example how to use run the compose files by running the following command: `d
 
 ### [compose-sandbox-env.yml](./Compose/pgsql/compose-sandbox-env.yml)
 
-Provides an implementation of a sandbox environment with the ODS/API, Sandbox Admin, and SwaggerUI behind nginx. The databases are installed on one instance of PostgreSQL 11.
+Provides an implementation of a sandbox environment with the ODS/API, Sandbox Admin, and SwaggerUI behind nginx. The databases _EdFi_Admin_ and _EdFi_Security_ are are installed on one instance of PostgreSQL 11. The _EdFi_Ods_ sandboxes and templates are installed on a separate instance of PostgreSQL 11.
```
Comment (tbmeador-dlp) [inline README.md:13]:
```suggestion
Provides an implementation of a sandbox environment with the ODS/API, Sandbox Admin, and SwaggerUI behind nginx. The databases _EdFi_Admin_ and _EdFi_Security_ are installed on one instance of PostgreSQL 11. The _EdFi_Ods_ sandboxes and templates are installed on a separate instance of PostgreSQL 11.
```

File: README.md [hunk 3]
```diff
@@ -10,11 +10,11 @@ An example how to use run the compose files by running the following command: `d
...
 
 ### [compose-shared-instance-env.yml](./Compose/pgsql/compose-shared-instance-env.yml)
 
-Provides an implementation of a shared instance environment of the ODS/API and Admin App behind nginx. The databases _EdFi_Admin_ and _EdFi_Security_ are are installed on one instance of PostgreSQL 11. The _EdFi_Ods_ database and the minimal template are installed on a separate instance of PostgreSQL 11.
+Provides an implementation of a shared instance environment of the ODS/API and Admin App behind nginx. The databases _EdFi_Admin_ and _EdFi_Security_ are are installed on one instance of PostgreSQL 11. The _EdFi_Ods_ database and minimal template are installed on a separate instance of PostgreSQL 11.
```
Comment (tbmeador-dlp) [inline README.md:17]:
```suggestion
Provides an implementation of a shared instance environment of the ODS/API and Admin App behind nginx. The databases _EdFi_Admin_ and _EdFi_Security_ are installed on one instance of PostgreSQL 11. The _EdFi_Ods_ database and minimal template are installed on a separate instance of PostgreSQL 11.
```

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]

**Pull request merged**